go/cmd/dolt/commands/sql.go

```diff
@@ -155,15 +169,46 @@ func (cmd SqlCmd) Exec(ctx context.Context, commandStr string, args []string, dE
 	saveMessage := apr.GetValueOrDefault(messageFlag, "")
 	saveName := apr.GetValueOrDefault(saveFlag, "")
 
-	err := validateSqlArgs(apr)
-	if err != nil {
-		return HandleVErrAndExitCode(errhand.VerboseErrorFromError(err), usage)
+	_, forceBatchMode := apr.GetValue(batchFlag)
+
+	query, queryOK := apr.GetValue(queryFlag)
+	if !queryOK {
```

Comment:
You already validated this in the validate method, no need for this check here. Just use exOk directly

Reply:
this got changed when I addressed your other comments.